meinberlin/assets/scss/components/_control-bar.scss

```diff
@@ -53,3 +42,79 @@
     z-index: 1;
     top: -1em;
 }
+
+.control-bar__search {
+    border-bottom: 1px solid $grey-dark;
+    margin-bottom: 1.25em;
+    padding-bottom: 0.25em;
+
+    @media (min-width: #{$breakpoint-palm + 0.0625rem}) {
+        padding-bottom: 1.25em;
+    }
+}
+
+.control-bar__filter-btn {
+    margin-top: 0.875em;
+
+    i {
+        margin-right: 0.625em;
+    }
+}
+
+.control-bar__bottom {
+    border-top: 1px solid $grey-dark;
+    margin-top: .875em;
+    padding-top: 1.5em;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.5em;
+
+    @media screen and (min-width: #{$breakpoint-palm + 0.0625rem}) {
+        flex-direction: row;
+        justify-content: space-between;
+        gap: 0;
+    }
+
+    @media screen and (max-width: $breakpoint-palm) {
+        .button {
+            width: 100%;
+        }
+    }
+}
+
+.control-bar__results {
+    font-weight: 700;
+}
+
+.control-bar__meta {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.875em;
+
+    @media screen and (min-width: #{$breakpoint-tablet + 0.0625rem}) {
+        flex-direction: row;
+        justify-content: space-between;
+    }
+}
+
+.control-bar__meta__left {
+    margin-bottom: 1.875em;
+
+    @media screen and (min-width: #{$breakpoint-tablet + 0.0625rem}) {
+        margin-bottom: 0;
+    }
+}
+
+.control-bar__list-map-switch {
+    display: flex;
+    align-items: center;
+
+    @media screen and (max-width: $breakpoint-tablet) {
+        width: 100%;
+
+        .button {
+            flex: 1 1 100%;
+        }
+    }
```

Comment:
some of these styles might be set by using the faceting form container classes, have sent a link